public/js/app.js

```diff
@@ -0,0 +1,110 @@
+// Call function getNextMove(player, coord)
+// player = 'computer' or 'human'
+// coord is an element in [0,1,2,3,4,5,6,7,8]
+// The function returns an element in [0,1,2,3,4,5,6,7,8]
+// as the computer's next move, 'computer', 'human', or 'tie'
+
+computerMoves = []
+humanMoves = []
+board = []
+wins = [[0,1,2],[3,4,5],[6,7,8],[0,3,6],[1,4,7],[2,5,8],[0,4,8],[2,4,6]]
+
```

Comment:
If the UI calls getNextMove, make you return an object so it will be easy to add more things if need be later on.
{
state: 'tie'
}

or
{
  move: <comp move>
}